packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -135,7 +135,9 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       connectorsAzureBlobStorage: `${ENTERPRISE_SEARCH_DOCS}connectors-azure-blob.html`,
       connectorsClients: `${ENTERPRISE_SEARCH_DOCS}connectors.html#connectors-build`,
       connectorsConfluence: `${ENTERPRISE_SEARCH_DOCS}connectors-confluence.html`,
-      connectorsContentExtraction: `${ENTERPRISE_SEARCH_DOCS}connectors-content-extraction.html`,
+      connectorsDropbox: `${ENTERPRISE_SEARCH_DOCS}connectors-content-extraction.html`,
```

Comment:
Looks like `Dropbox` and `contentExtraction` are mixed up here

Reply:
fixed